boot/bootchain.go

```diff
@@ -35,23 +35,23 @@ import (
 )
 
 // TODO:UC20 add a doc comment when this is stabilized
```

Comment:
btw. we can probably address this now?

Reply:
I think we might change this very soon. Because it does not make much sense for fde hook. But we do need them now for it.